core/shared/src/main/scala/matryoshka/Recursive.scala

```diff
@@ -171,7 +171,7 @@ trait Recursive[T] extends Based[T] { self =>
   def gcataZygo[W[_]: Comonad, A, B]
     (t: T)
     (k: DistributiveLaw[Base, W], f: GAlgebra[W, Base, B], g: GAlgebra[(B, ?), Base, A])
-    (implicit BF: Functor[Base], BU: Unzip[Base]) =
+    (implicit BF: Functor[Base]/*, BU: Unzip[Base]*/): A =
```

Comment:
I left these in because I was unsure if there was some other rationale for asking for the implicit. I can remove the commented code as long as you approve.

Reply:
@alissapajer @wemrysi can you think of any reason of these implicits being there but not being used?

Reply:
I'd guess that they're left over from a time when they were needed, or maybe they were never needed. If it compiles without them, we can delete them.

Reply:
@rintcius @alissapajer I pushed another commit which deletes them.